lmfit/lineshapes.py

```diff
@@ -145,6 +145,24 @@ def moffat(x, amplitude=1, center=0., sigma=1, beta=1.):
     return amplitude / (((x - center)/max(tiny, sigma))**2 + 1)**beta
 
 
+def pearson4(x, amplitude=1.0, center=0.0, sigma=1.0, expon=1.0, skew=0):
+    """Return a Pearson4 lineshape.
+
+    Using the Wikipedia definition:
```

Comment:
can you please add the link to the Wikipedia where the lineshape is defined (I know we don't have that for the Pearson7 either, and it would probably be a good addition to add that as well). We have the links present for most other lineshapes

Reply:
I think there may not be a single link to the equation... but a line to the main Pearson distribution page is probably good.